IPs/RSKIP434.md

```diff
@@ -0,0 +1,86 @@
+---
+rskip: 434
+title: Bridge Bitcoin block chainwork up to 12 Bytes
+created: 26-Jun-24
+author: JD,JZ,MI
+purpose: Sec
+layer: Core
+complexity: 1
+status: Draft
+description: Change in Bridge Bitcoin storage to accept blocks with chainwork serialized up to 12 Bytes
+---
+
+|RSKIP          |434           |
+| :------------ |:-------------|
+|**Title**      |Bridge Bitcoin block chainwork up to 12 Bytes |
+|**Created**    |26-Jun-24 |
+|**Author**     |JD,JZ,MI |
+|**Purpose**    |Sec |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+The Rootstock Bridge functions as an SPV (Simplified Payment Verification) node. This means it operates by receiving block headers from the Bitcoin blockchain through external entities known as relayers. The bridge verifies the validity of Bitcoin transactions by requesting users to submit Merkle proofs, which follow Bitcoin's SPV technique. This setup enables the bridge to confirm that funds have been locked on the Bitcoin blockchain without needing to trust the relayers fully, as the verification is done through cryptographic proof.
+
+This mechanism is crucial for maintaining a trust-minimized operation, ensuring that users can independently verify the locking of funds on Bitcoin, which is essential for the security and functionality of cross-chain transactions between Bitcoin and Rootstock.
+
+Chain work in Bitcoin refers to the cumulative computational effort that has been expended to build the blockchain. It is a measure of the total work done by miners to create and validate blocks, ensuring the security and integrity of the blockchain. The chain work is currently stored in a signed 12 Bytes element in the Block store.
+
+As of block #849,138 chainWork has reached a value of 800028897aa1dce163ead2cf which exceeds the capacity of the 12 Bytes storage format.
```

Comment:
```suggestion
As of block #849,138, chainWork has reached a value of `0x800028897aa1dce163ead2cf`, which exceeds the capacity of the 12-byte storage format.
```

Reply:
agree with natha,
also #849_138 instead of #849,138